libs/sdk-ui-dashboard/src/model/commandHandlers/rootCommandHandler.ts

```diff
@@ -32,21 +32,26 @@ import { drillToInsightHandler } from "./drill/drillToInsightHandler";
 import { drillToCustomUrlHandler } from "./drill/drillToCustomUrlHandler";
 import { drillToAttributeUrlHandler } from "./drill/drillToAttributeUrlHandler";
 import { drillToDashboardHandler } from "./drill/drillToDashboardHandler";
+import { changeFilterContextSelectionHandler } from "./filterContext/changeFilterContextSelectionHandler";
+import { drillToLegacyDashboardHandler } from "./drill/drillToLegacyDashboardHandler";
+import { logUserInteractionHandler } from "./logUserInteraction/logUserInteractionHandler";
 
 const DefaultCommandHandlers: {
-    [cmd in DashboardCommands["type"]]?: (...args: any[]) => SagaIterator<any>;
+    [cmd in DashboardCommands["type"]]?: (...args: any[]) => SagaIterator<any> | any;
 } = {
     "GDC.DASH/CMD.LOAD": loadDashboardHandler,
     "GDC.DASH/CMD.SAVE": unhandledCommand,
     "GDC.DASH/CMD.SAVEAS": unhandledCommand,
     "GDC.DASH/CMD.RESET": unhandledCommand,
     "GDC.DASH/CMD.RENAME": unhandledCommand,
-    "GDC.DASH/CMD.DATE_FILTER.CHANGE_SELECTION": changeDateFilterSelectionHandler,
-    "GDC.DASH/CMD.ATTRIBUTE_FILTER.ADD": addAttributeFilterHandler,
-    "GDC.DASH/CMD.ATTRIBUTE_FILTER.REMOVE": removeAttributeFiltersHandler,
-    "GDC.DASH/CMD.ATTRIBUTE_FILTER.MOVE": moveAttributeFilterHandler,
-    "GDC.DASH/CMD.ATTRIBUTE_FILTER.CHANGE_SELECTION": changeAttributeFilterSelectionHandler,
-    "GDC.DASH/CMD.ATTRIBUTE_FILTER.SET_PARENT": setAttributeFilterParentHandler,
+    "GDC.DASH/CMD.LOG_USER_INTERACTION": logUserInteractionHandler,
```

Comment:
Should we namespace this? Like `GDC.DASH/CMD.USER_INTERACTION.LOG`? So that we can then do some other LOG types like for example `GDC.DASH/CMD.FILTER_CONTEXT.CHANGE_SELECTION.LOG`?

Reply:
I'm not sure. Most of the time, logging should be just about listening relevant events (in your use case `GDC.DASH/CMD.FILTER_CONTEXT.CHANGE_SELECTION`). Some interactions are not covered by the existing events and does not have place in our model - this is reason for this event. Maybe we should omit "log" word at all, and inform just about "user interactions". What do you think?

Reply:
Good point. I agree that logging should "live" in KD, so maybe go with your suggestion and remove the `_LOG` part.